src/main/java/core/basesyntax/strategy/FruitOperationHandler.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dto.FruitRecordDto;
+
+public interface FruitOperationHandler {
+    String EXCEPT_LETTERS = "[^A-z]";
+    int apply(FruitRecordDto fruitRecordDto);
+
+    default void checkQuantity(Integer quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("Negative value of the number of fruits");
+        }
+    }
+
+    default void checkFruitName(String fruitType) {
+        if (fruitType.equals("null")
```

Comment:
why do you compare type with particular string "null"?

Reply:
I agree, this is an unnecessary check